agent/hcp/client/metrics_client.go

```diff
@@ -24,10 +24,12 @@ const (
 	defaultStreamTimeout = 15 * time.Second
 
 	// Retry config
-	// TODO: Evenutally, we'd like to configure these values dynamically.
+	// TODO: Eventually, we'd like to configure these values dynamically.
 	defaultRetryWaitMin = 1 * time.Second
 	defaultRetryWaitMax = 15 * time.Second
-	defaultRetryMax     = 4
+	// defaultRetryMax is set to 0 to turn off retry functionality, until dynamic configuration is possible.
+	// This reduces load on the Telemetry Gateway for now.
```

Comment:
Pedantic comment to say that the concern is not about baseline load but about 4X spikes in load that cause or exacerbate server-side issues

Reply:
yep sounds good, I'll clarify, wasn't sure how to phrase it 😓